frontend/src/composables/usePageTitle.js

```diff
@@ -0,0 +1,22 @@
+import { ref, onMounted, watch } from 'vue';
+import { useRoute } from 'vue-router';
+
+export default function usePageTitle(pageTitle) {
+  const defaultTitle = "AIMAAS"; // used when no title is defined for given route
+  const route = useRoute();
+  const title = ref(defaultTitle);
+
+  if (pageTitle) {
+  document.title = pageTitle || route.meta.title || defaultTitle;
+}
```

Comment:
I think, these lines could be superfluous. If I comment them out, it still works as expected.

```suggestion
```

Reply:
Thank you for changing this.

On a side note: You did not have to rebase and force-push. An additional commit (as created when clicking on "Commit suggestion" would have been fine, too. :slightly_smiling_face: